src/ocrd_network/processing_server.py

```diff
@@ -736,9 +736,7 @@ async def run_workflow(
         self.validate_tasks_agents_existence(processing_tasks, agent_type)
 
         page_ids = get_page_ids_list(self.log, mets_path, page_id)
-
-        # TODO: Reconsider this, the compact page range may not always work if the page_ids are hashes!
-        compact_page_range = f"{page_ids[0]}..{page_ids[-1]}"
+        compact_page_range = ','.join(page_ids)
```

Comment:
That is a good fix for non-integer page_ids. However, for integer page_ids it will bloat the database with unnecessarily big `compact_page_range` values. Instead of `PHYS_0001..PHYS_1234`, the value will be a very big string `PHYS_0001,PHYS_0002,PHYS_0003...PHYS_1234`. Is there maybe a way to keep it as it is for integer page_ids, and join for non-integer ones? For the sake of optimization.

Reply:
@bertsky, if that works, yes!